--- a/pub/index.php
+++ b/pub/index.php
@@ -8,6 +8,15 @@
 
 use Magento\Framework\App\Bootstrap;
 
+header('Access-Control-Allow-Origin: *');
+header('Access-Control-Allow-Methods: *');
+header('Access-Control-Allow-Headers: DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Content-Range,Range,Authorization,Store');
+
+if ($_SERVER['REQUEST_METHOD'] == 'OPTIONS') {
+    http_response_code(200);
+    exit;
+}
+
 try {
     require __DIR__ . '/../app/bootstrap.php';
 } catch (\Exception $e) {
